oracle/oracle/validators/eth1.py

```diff
@@ -28,7 +29,36 @@ async def select_validator(
         query=OPERATORS_QUERY,
         variables=dict(block_number=block_number),
     )
-    operators = result["operators"]
+    operators = result["operators"][:1]
+    result: Dict = await execute_sw_gql_query(
+        network=NETWORK,
+        query=LAST_VALIDATORS_QUERY,
+        variables=dict(block_number=block_number),
+    )
+    last_validators = result["validators"]
+
+    if last_validators:
+        last_operator_id = last_validators[0]["operator"]["id"]
+        index = None
+        for i, operator in enumerate(operators):
+            if operator["id"] == last_operator_id:
+                index = i
+                break
+        if index is not None:
+            operators.append(operators.pop(index))
+
+    if NETWORK_CONFIG["ORACLE_IGNORE_STAKEWISE_OPERATOR"]:
+        index = None
+        for i, operator in enumerate(operators):
+            if (
+                Web3.toChecksumAddress(operator["id"])
+                == NETWORK_CONFIG["ORACLE_STAKEWISE_OPERATOR"]
+            ):
+                index = i
+                break
+        if index is not None:
+            operators.append(operators.pop(index))
```

Comment:
It looks too complex to me.
I suggest we replace `orderBy: validatorsCount` with `orderBy: id` in the OPERATORS_QUERY, sort operators as [operators after last_operator_id, last_operator_id, operators before last_operator_id, stakewise operator], and iterating the same way we did before.